architecture/infrastructure_components/web_console.adoc

```diff
@@ -50,6 +50,12 @@ whitelist that host name by specifying the `--cors-allowed-origins` option
 on `openshift start` or from the related
 xref:../../install_config/master_node_configuration.adoc#master-configuration-files[master
 configuration file parameter `corsAllowedOrigins`].
+
+[NOTE]
+====
+The `corsAllowedOrigins` parameter is fully under the control of the
+configuration field. No pinning or escaping is done to the value.
+====
```

Comment:
it would probably be good to give an example of how you to pin a hostname and escape dots:

```yaml
corsAllowedOrigins:
- (?i)//my\.subdomain\.domain\.com(:|\z)
```

Explanations:
* `(?i)` makes it case-insensitive
* `//` pins to the beginning of the domain (matches the double slash following http: or https:)
* `\.` escape dots in the domain name
* `(:|\z)` matches the end of the domain name (`\z`) or a port separator (`:`)